frontend/packages/core/src/Feedback/tooltip.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { TooltipProps as MuiTooltipProps } from "@material-ui/core";
+import { Tooltip as MuiTooltip } from "@material-ui/core";
+
+const BaseTooltip = ({ className, ...props }: MuiTooltipProps) => (
```

Comment:
the tooltip component accepts a popper component, so initially I was styling the tooltip by targeting it through the popper. however, things got complicated when I wanted to make the maxWidth prop-based as  I'd have to pass the prop from tooltip --> popper component. I was getting various errors and it felt messy. 

Found this pattern here https://stackoverflow.com/questions/59934683/style-material-ui-tooltip-using-emotion-styled and we also do something similar for our Select component https://github.com/lyft/clutch/blob/main/frontend/packages/core/src/Input/select.tsx#L61-L78